eng/ci/templates/jobs/run-integration-tests-linux.yml

```diff
@@ -0,0 +1,44 @@
+jobs:
+- job: RunIntegrationTests_Linux
+  displayName: '[Linux] Run Integration Tests'
+
+  pool:
+    name: 1es-pool-azfunc
+    image: 1es-ubuntu-22.04
+    os: linux
+
+  steps:
+  - task: NuGetAuthenticate@1
+    displayName: NuGet Authenticate
+
+  - template: /eng/ci/templates/steps/install-dotnet.yml@self
+
+  - task: DotNetCoreCLI@2
+    displayName: Build DotNetWorker
+    inputs:
+      command: build
+      arguments: '-c Release'
+      projects: DotNetWorker.sln
+
+  - pwsh: |
+      $useIntegrationTestingCoreTools = $null
+      if (-not ([bool]::TryParse($env:USECORETOOLSBUILDFROMINTEGRATIONTESTS, [ref] $useIntegrationTestingCoreTools)))
+      {
+        throw "UseCoreToolsBuildFromIntegrationTests can only be set to True or False. Current value is set to $env:USECORETOOLSBUILDFROMINTEGRATIONTESTS"
+      }
+      ./setup-e2e-tests.ps1 -DotnetVersion 'net7' `
+                            -UseCoreToolsBuildFromIntegrationTests:$useIntegrationTestingCoreTools `
+                            -SkipCosmosDBEmulator `
+                            -SkipBuildOnPack
+    displayName: "Setup E2E tests"
+    env:
+      CORE_TOOLS_URL: $(CORE_TOOLS_URL)
```

Comment:
Is this needed? Powershell will already be inheriting that environment variable.

Reply:
Will remove it and see what happens